src/platform/site-wide/sass/shame.scss

```diff
@@ -636,3 +636,92 @@ div[data-widget-type="events"] {
   }
 }
 /* ^^^ DO NOT REMOVE w/o DS REVIEW: solves !important conflict*/
+
+/* ~~~~~ DO NOT REMOVE w/o DS REVIEW: Temporary fixes to enable core.scss swap ~~~~~~ */
```

Comment:
For the sake of keeping the mental model relatively simple, I'm just moving our grid classes into shame for the time being. I know I know, we want to get rid of shame.scss eventually. This makes testing easier. These grid classes will 100% be moved to css-library when we have the capacity to update all of grid.